src/v2/Apps/Auction/Components/BidForm.tsx

```diff
@@ -85,10 +85,7 @@ export const BidForm: React.FC<Props> = ({
   return (
     <Box maxWidth={550}>
       <Formik<FormValues>
-        initialValues={{
```

Comment:
Just wanna confirm that this commit is just linting/formatting – I think it is

Reply:
This is formatting but I think this is mostly me extracting the `initialValues`.

Reply:
Oh I see what you mean now. Yeah this is just formatting. The `-w` flag might be helpful to push out whitespace changes.